packages/ui/src/2_molecules/Pagination/Pagination.test.tsx

```diff
@@ -0,0 +1,31 @@
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import React from 'react';
+
+import { Pagination } from './Pagination';
+
+describe('Pagination', () => {
+  it('renders Pagination', () => {
+    const gotoPage = jest.fn();
+    const { findAllByRole } = render(
+      <Pagination setPage={gotoPage} page={0} totalItems={20} />,
+    );
+    expect(findAllByRole('svg[data-icon="arrow-back"]')).toBeDefined();
+  });
+
+  it('does not allow to click Previous without clicking Next first', () => {
```

Comment:
This label is misleading because you don't test what it says, you just click "Next". Rename it and add a test that actually tests the described behaviour.